package.json

```diff
@@ -173,81 +178,99 @@
     "electron": "./node_modules/.bin/electron"
   },
   "devDependencies": {
+    "@babel/core": "7.7.2",
+    "@babel/plugin-proposal-class-properties": "7.7.0",
+    "@babel/plugin-proposal-decorators": "7.7.0",
+    "@babel/plugin-proposal-do-expressions": "7.6.0",
+    "@babel/plugin-proposal-export-default-from": "7.5.2",
+    "@babel/plugin-proposal-export-namespace-from": "7.5.2",
+    "@babel/plugin-proposal-function-bind": "7.2.0",
+    "@babel/plugin-proposal-function-sent": "7.7.0",
+    "@babel/plugin-proposal-json-strings": "7.2.0",
+    "@babel/plugin-proposal-logical-assignment-operators": "7.2.0",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "7.4.4",
+    "@babel/plugin-proposal-numeric-separator": "7.2.0",
+    "@babel/plugin-proposal-optional-chaining": "7.6.0",
+    "@babel/plugin-proposal-pipeline-operator": "7.5.0",
+    "@babel/plugin-proposal-throw-expressions": "7.2.0",
+    "@babel/plugin-syntax-dynamic-import": "7.2.0",
+    "@babel/plugin-syntax-import-meta": "7.2.0",
+    "@babel/plugin-transform-classes": "7.7.0",
+    "@babel/plugin-transform-react-constant-elements": "7.6.3",
+    "@babel/plugin-transform-react-inline-elements": "7.2.0",
+    "@babel/polyfill": "7.7.0",
+    "@babel/preset-env": "7.7.1",
+    "@babel/preset-react": "7.7.0",
+    "@babel/register": "7.7.0",
+    "@babel/runtime-corejs2": "7.7.2",
+    "@jest-runner/electron": "2.0.2",
     "@ronomon/direct-io": "2.4.3",
     "aws-sdk": "2.472.0",
-    "babel-core": "6.26.3",
+    "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "10.0.3",
-    "babel-loader": "7.1.5",
-    "babel-plugin-add-module-exports": "1.0.0",
+    "babel-jest": "24.9.0",
+    "babel-loader": "8.0.6",
+    "babel-plugin-add-module-exports": "1.0.2",
     "babel-plugin-dev-expression": "0.2.2",
     "babel-plugin-module-resolver": "3.2.0",
     "babel-plugin-rewire": "1.2.0",
-    "babel-plugin-transform-class-properties": "6.24.1",
-    "babel-plugin-transform-define": "1.3.1",
-    "babel-plugin-transform-es2015-classes": "6.24.1",
-    "babel-plugin-webpack-loaders": "0.9.0",
-    "babel-polyfill": "6.26.0",
-    "babel-preset-env": "1.7.0",
-    "babel-preset-react": "6.24.1",
-    "babel-preset-react-hmre": "1.1.1",
-    "babel-preset-react-optimize": "1.0.1",
-    "babel-preset-stage-0": "6.24.1",
-    "babel-register": "6.26.0",
-    "babel-runtime": "6.26.0",
+    "babel-plugin-transform-define": "2.0.0",
+    "babel-plugin-transform-react-remove-prop-types": "0.4.24",
     "babili-webpack-plugin": "0.1.2",
-    "chai": "4.1.2",
+    "chai": "4.2.0",
     "concurrently": "5.0.0",
     "cross-env": "6.0.3",
-    "css-loader": "0.28.11",
+    "css-loader": "3.2.0",
     "devtron": "1.4.0",
     "difflet": "1.0.1",
     "drivelist": "6.4.3",
     "electron": "3.1.1",
-    "electron-builder": "21.2.0",
+    "electron-builder": "22.1.0",
```

Comment:
This is still marked as a pre-release, so maybe we should hold off on this if it's not required to make things work?

Reply:
Yeah, I'm fine with that. I was just trying for as many small updates as I could get and presumed it wasn't a pre-release since it was a final version number (I didn't look too closely).